runestone/video/video.py

```diff
@@ -214,37 +210,18 @@ class Youtube(IframeVideo):
    """
     html = '''
     <div class="runestone" style="margin-left: auto; margin-right:auto">
-    <div id="%(divid)s_vid"></div>
+	<iframe id="%(divid)s_vid" type="text/html" width="%(width)u" height="%(height)u"
+  src="https://www.youtube.com/embed/%(video_id)s?autoplay=0"
+  frameborder="0"></iframe>
     </div>
-    <script id="%(divid)s_script">
```

Comment:
Unless there is more to come on this, I can't accept it as is.   For grading and analytics purposes we need to be able to capture events.  That is why I went this route using the youtube api.  

I'm pretty usre the solution I outlined under the TODO will solve the problem and preserve our logging.

Reply:
OK, I will look into implementing a version that preserves logging.

Reply:
Awesome!  Thanks